### ISSUE TITLE
A11Y: Focus rings should be used for all interactive elements

### ISSUE THREAD

**Issue by paul-geoghegan** [canonical/ubuntu-desktop-provision#1475]
Currently we have focusrings disabled in our Yaru theme so that they could be consistant with the default Flutter theme. We should revert this and wrap any custom widgets in the Yaru Focus Boarder widget in order to have better focus state behavior and improve a11y for sighted keyboard users.
